### ISSUE TITLE
index.js:1205 Uncaught Error: SecurityError: An insecure SockJS connection may not be initiated from a page loaded over HTTPS

### ISSUE THREAD

**Issue by hungdev** [lxieyang/chrome-extension-boilerplate-react#149]
I get the error. please tell me how to fix it?
<img width="563" alt="image" src="https://user-images.githubusercontent.com/12009276/212457314-ce041675-fef9-4ffa-88a6-522849b81d95.png">

**Comment by ReAlign** [comment]
+1

**Comment by ReAlign** [comment]
https://webpack.js.org/configuration/dev-server/#devserverhttps

**Comment by ReAlign** [comment]
+1

**Comment by ReAlign** [comment]
Maybe:

<img width="692" alt="Xnip2023-01-14_20-01-24" src="https://user-images.githubusercontent.com/25849349/212470684-f5d5cd7f-76a0-46fb-91d9-605dfa3d9e8b.png">

**Comment by anass-qalqul** [comment]
yeah, it is a blocking error

**Comment by lxieyang** [comment]
Hmm, I haven't been able to reproduce. Would you try adding `https: false` to the webserver config?

**Comment by ReAlign** [comment]
content script error, when open a third-part page:

<img width="1724" alt="image" src="https://user-images.githubusercontent.com/25849349/212527311-2203648a-3acc-4fc0-abec-dda99633c709.png">

**Comment by ReAlign** [comment]
Anyone help?

**Comment by kalyncoose** [comment]
This is definitely an issue with the WebPack or Web Dev Server and I am not proficient enough at either to know how to fix it.

![websocket-error](https://user-images.githubusercontent.com/65800415/213344078-bd8f65a1-1a45-4bc4-ad22-1e99aa445638.PNG)

However I cloned [@octohedron](https://github.com/octohedron/chrome-extension-boilerplate-react)'s updated fork with a completely custom web server. I was able to install, unpack, and try it out and it had no web socket errors and the content scripts loaded without issues and I could see the console logs it does from the template.

**Comment by amitava82** [comment]
Yes this is an issue with content script not loading on any page due to this error

**Comment by hadnet** [comment]
This error still happens. Any update to fix this issue?

**Comment by uri3000** [comment]
Content script not working due to this... it kind of makes the whole boilerplate unusable.  any help?

**Comment by kalyncoose** [comment]
@hadnet @uri3000 [Use this fork](https://github.com/octohedron/chrome-extension-boilerplate-react) instead, it has a better and working dev server.

**Comment by shawnvoss** [comment]
In webserver.js, change `https` property to `true` **. This will get things working, even though you will still see errors on the console. Probably need to [configure a local cert](https://webpack.js.org/configuration/dev-server/#devserverserver) to completely silence this, but at least the content script will be loading/running.

** Better yet, replace `https: true` with `server: 'https'`, as the `https` [property is deprecated](https://webpack.js.org/configuration/dev-server/#devserverhttps).

**Comment by Suket-shah** [comment]
Was anyone able to configure the local cert to silence the error?

** Edit: by making the change @shawnvoss suggests, the error no longer appears
